packages/near-membrane-base/src/membrane.ts

```diff
@@ -570,6 +564,21 @@ export function createMembraneMarshall() {
             return getTransferablePointer(value);
         }
 
+        function getUnbrandedTag(value: any): string | undefined {
+            if (
+                typeof value === 'object' &&
+                value !== null &&
+                !isArrayOrNotOrThrowForRevoked(value)
```

Comment:
keep in mind that this line might throw if value is a revoked proxy.

Reply:
wrapped in a try-catch